### PR TITLE
Moving tokenizer to DbotPreprocessTextData

### PR DESCRIPTION
**Pull request by lperryp**
## Description
Moving tokenization logic to DbotPreprocessTextData 
Changing GetMLModelEvaluation to python3
This is a continuation of : https://github.com/demisto/content/pull/9536

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Base/Scripts/DBotPreprocessTextData/DBotPreprocessTextData.py [hunk 1]
```diff
@@ -84,43 +278,47 @@ def remove_line_breaks(text):
...
 
 
-def pre_process_nlp(text_data):
-    res = demisto.executeCommand('WordTokenizerNLP', {
-        'value': json.dumps(text_data),
-        'isValueJson': 'yes',
-        'tokenizationMethod': demisto.args()['tokenizationMethod'],
-        'language': demisto.args()['language']
-    })
+def pre_process_nlp(text, seed):
```
Comment (erezh31) [inline Packs/Base/Scripts/DBotPreprocessTextData/DBotPreprocessTextData.py:304]: you can rename this function, it's not nlp anymore, it's more generic - you can get the tokenizer method here as arrgument

File: Packs/Base/Scripts/DBotPreprocessTextData/DBotPreprocessTextData.py [hunk 2]
```diff
@@ -1,15 +1,209 @@
 # pylint: disable=no-member
 from sklearn.feature_extraction.text import TfidfVectorizer
-
-import demistomock as demisto
-from CommonServerPython import *
 from CommonServerUserPython import *
-
 import pickle
 import uuid
+from CommonServerPython import *
```
Comment (erezh31) [inline Packs/Base/Scripts/DBotPreprocessTextData/DBotPreprocessTextData.py:6]: bring this import first

**Review by erezh31 — APPROVED** [review]
approved with small comments

**Comment by lperryp** [comment]
labeling "docs-approved" as RNs were checked here https://github.com/demisto/content/pull/9536

**Pull request merged**